### ISSUE TITLE
Amsterdam contribution to the FAIR Expert Group

### ISSUE THREAD

**Issue by WouterLos** [FAIR-EG/consultation#28]
[//]: # "==Do not write above this line==
Instructions for posting issues: 
(1) Review what is already there. Perhaps a comment to an existing issue would be more appropriate than opening a new one?
(2) Write your post below using Markdown (as per https://guides.github.com/features/mastering-markdown/ ) or just plain text.
(3) Don't worry about these introductory lines - you can leave or delete them, as they won't display anyway (you can check this via Preview).
(4) Hit the 'Submit new issue' button.
==Write below this line=="
**The Amsterdam Economic Board and the Amsterdam Science Park**
contribution to the FAIR data Expert Group Call for Contributions

With this letter the Amsterdam Metropolitan Area would like to set out its vision on FAIR data and offer its support in turning the FAIR data principles into an operational reality. 
First of all, we fully support and compliment the data related efforts of the European Council and the European Commission. The Digital Single market, the European Cloud Initiative, EOSC and FAIR data initiatives are all part of a roadmap to innovation. Furthermore, we applaud the announced focus of the Estonian Presidency on a fifth freedom on free movement of data (https://www.eu2017.ee/news/insights/FreeMovementOfData). 

**Data-driven innovation in the Amsterdam metropolitan area**
Data-driven innovation is crucial for the Amsterdam Metropolitan Area in a variety of sectors such as mobility, health, sustainability, and of course advanced research. A strong signal from the Amsterdam municipality was the decision to endorse the principle of open data for all relevant information. This key choice has its governance implications, but this signal is driving change to a new culture of benefitting from open data, not only for public services but also for the private sector and the public at large.
Stakeholders in the Amsterdam region are cooperating to create an open cloud environment based on FAIR data. This movement is not restricted to the scientific communities and their data but also includes other public and private sectors. We consider the FAIR principals crucial for seamless and trusted use of data across disciplines and organisations. The data sharing initiatives in the Amsterdam region are mostly bottom-up and community driven. Drivers for these initiatives are the emerging new business models in the open cloud, and the need to agree on organising the related   trust in the concerned communities. All initiatives are cooperating with similar developments in the Netherlands and abroad, which positions the Amsterdam region as an area of expertise and as model region for other regions and countries.
Examples of the initiatives (with stakeholders in italics) in the Amsterdam Metropolitan Area are:
-	The Commit2Data Hub connecting _business and science_ with services to process data according to agreed conditions (https://www.dutchdigitaldelta.nl/en/commit2data);
-	Data Hub Arena, a place for _entrepreneurs, governmental organisations and researchers_ to share data and create data-driven innovations for entertainment, media, mobility, safety and sustainability (https://www.bigdata-alliance.org/news/amsterdam-arena-opens-first-big-data-hub-netherlands/); 
-	DECODE, a EU H2020 project, to increase the digital sovereignty of _European citizens_ by enabling them to produce, access and control their data and exchange contextualised information in real-time, and in a confidential, and scalable manner (https://decodeproject.eu);  
-	SCORE, an Interreg project of _cities in the North Sea Region_ including Amsterdam, collaborating to improve public service delivery based on smart, data-driven solutions. E.g. optimising the routing of garbage trucks based on waste location data, or guiding vehicles to available parking spots using sensor data and GPS (http://www.thenextsiliconvalley.com/2017/07/10/9445-nine-european-cities-to-develop-test-bed-for-smart-cities-data-sharing/).   

**Our recommendations**
The following paragraphs offer our contribution for the EC expert group on FAIR data, following the proposed sections in its call for contributions.

_Involve public and private partners (Section: concept- why FAIR?)_
The vision on FAIR data is currently mainly regarded for research communities. However, we also see value of the FAIR principles for other public and private communities. A new economy is emerging, offering products and services with added value on shared data. A prerequisite is that all involved stakeholders have trust in the full cycle from raw to processed data. Data are only understandable in the context of how they were produced, and as such also software code, virtual environments, sensors, machines (Internet of Things) can be regarded as FAIR data and services Here the FAIR principles offer a common baseline for organising trust. Apart from being Findable, Accessible, Interoperable, and Re-usable, the data also have to be machine-understandable allowing for automatic processing. In addition, data should be legally interoperable with a limited set of machine-readable license formats, whilst guaranteeing digital trust among all parties involved, including citizens. Stakeholders in the Amsterdam metropolitan region are putting effort in researching and testing such approaches.

_Apply a bottom-up community strategy (Section: research data culture)_
While we regard the required culture not restricted to the research communities, we observe that culture change is mainly driven by opportunities for creating new knowledge, new services, new business models, or new strategic partnerships. It is conditional to implement the supporting software tools that will facilitate a FAIR policy.  For example to support the direct upload of new data in correct formats, and tools to (re)use data in applications. Not until such tools will offer an accepted default mode of work, a new data culture is expected to expand in time. It has been suggested to define “Rules of Engagement”, being a kind of Charters on issues such as quality, interoperability, privacy, and safety. Regional, national or sectorial supervisory bodies established by cooperating actors in the landscape of providers could develop the mechanisms to enforce such rules. Such a bottom-up community strategy, rather than top-down governance, will facilitate adaptive developments.

_Establish local support teams (Section: facilitating change)
Above, already some facilitating mechanisms are mentioned such as supporting tools, common Charters, and community supervision. Various mechanisms, standards and procedures will help communities to define a Roadmap towards organising trust. Promotion of national, regional or sectorial networks with adequate support in order to implement the mechanisms is crucial for adopting new working processes as well as new opportunities in science, public sector, business and society at large. Stakeholders in the Amsterdam Metropolitan Area will contribute to the call of the Netherlands and Germany to follow FAIR principles, and are considering plans to establish technical and legal support teams to assist developing networks in other countries. 

We’re looking forward to cooperate on turning the FAIR data principles into an operational reality.

Kind regards,

Nina Tellegen 						        Leo le Duc
Amsterdam Economic Board				Amsterdam Science Park
